src/Payments.Mvc/Controllers/FinancialAccountsController.cs

```diff
@@ -50,7 +53,7 @@ public FinancialAccountsController(ApplicationDbContext context, IFinancialServi
             }
 
 
-            if (team.Accounts.Where(a => a.IsActive && a.IsDefault).Count() != 1)
+            if (!team.Accounts.Any(a => a.IsActive && a.IsDefault))

```

Comment:
@jpknoll I had it this way to warn them if they had more that one default account selected.

Reply:
Okay. I think that should be different error messages though. I'll break it into two different if statements.